src/styles/stars.css

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(/src/images/star.svg);
```

Comment:
The path to the image should be relative to the compiled CSS file, not the source file. If this CSS is processed and ends up in a different directory, the path to the image will be incorrect. Make sure the path is correct after processing.